01-Introducing-Molecular-Dynamics/01-Molecular-Dynamics-Simulations.ipynb

```diff
@@ -138,7 +138,7 @@
    "outputs": [
```

Comment:
<p><code>so it can be reused</code>  to <code>so it can be used for multiple timesteps without recomputing neighbors</code> .</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Done.